notebooks/beginner.ipynb

```diff
@@ -614,19 +584,503 @@
    "source": [
     "k.expand.to_s"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SymEngine::RealDouble\n",
+    "\n",
+    "SymEngine::RealDouble can be constructed by converting any ruby Float into SymEngine"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "1.2"
+      ],
+      "text/plain": [
+       "#<SymEngine::RealDouble(1.2)>"
+      ]
```

Comment:
Is this difference between `text/html` and `text/plain` ok? As far as I can understand, IRuby prefers to output `text/html`, when finds it, and it completely hides the fact that we are working with SymEngine object.

BTW, I like to do HTML representation of such kind of objects [this way](https://render.githubusercontent.com/view/ipynb?commit=fd17c6b786e23ca58d532cfcf37103864da1ea07&enc_url=68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f6d6f6c796264656e756d2d39392f7265616c6974792f666431376336623738366532336361353864353332636663663337313033383634646131656130372f6e6f7465626f6f6b732f507265766965772e6970796e62&nwo=molybdenum-99%2Freality&path=notebooks%2FPreview.ipynb&repository_id=47512659#Entity) (look on blue-ish entities, and hover the mouse over them): basic text is as simple as possible, background+border shows "it is something more complex", tooltip shows `inspect` results.